src/Python/qsharp-core/qsharp/serialization.py

```diff
@@ -7,6 +7,11 @@
 # Licensed under the MIT License.
 ##
 
+try:
+    import numpy as np
+except ImportError:
```

Comment:
We should check, but it may be possible for importing numpy to raise an exception other than `ImportError` if NumPy has been installed, but is broken. In that case, we may still want to fail hard, but it may be worth catching that exception so to give a better error message.